src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html

```diff
@@ -131,21 +131,43 @@ <h3 class="inline">Results</h3>
 </ng-template>
 <ng-template #tpl let-element>
   <tr class="mat-row expandRow">
-    <td colspan="2"></td>
+    <td colspan="1"></td>
 
     <td class="linkList">
       <div class="linkListInner">
-        <div *ngFor="let manipulation of element.manipulations">
-          <a *ngIf="element.method === 'PathwayPredictor+DifferentialFVA'" [attr.href]="geneLink(manipulation)" target="_blank">
-            {{dispManipulation(manipulation)}}
-          </a>
-          <a *ngIf="element.method === 'PathwayPredictor+CofactorSwap'" [attr.href]="geneLink(manipulation)" target="_blank">
+        <div *ngFor="let manipulation of getManipulations(element.manipulations); let i = index;">
+          <div *ngIf="i < 10">
+            <a *ngIf="element.method === 'PathwayPredictor+DifferentialFVA'" [attr.href]="geneLink(manipulation)" target="_blank">
+              {{dispManipulation(manipulation)}}
+            </a>
+            <a *ngIf="element.method === 'PathwayPredictor+CofactorSwap'" [attr.href]="geneLink(manipulation)" target="_blank">
+            <span class="aligned-with-icon">
+              {{manipulation.from}}
+              <mat-icon class="arrow-icon">arrow_forward</mat-icon> {{manipulation.to}}
+            </span>
+            </a>
+          </div>
+          <div *ngIf="i >= 10" [hidden]="!showAllManipulations">
+            <a *ngIf="element.method === 'PathwayPredictor+DifferentialFVA'" [attr.href]="geneLink(manipulation)" target="_blank">
+              {{dispManipulation(manipulation)}}
+            </a>
+            <a *ngIf="element.method === 'PathwayPredictor+CofactorSwap'" [attr.href]="geneLink(manipulation)" target="_blank">
             <span class="aligned-with-icon">
               {{manipulation.from}}
               <mat-icon class="arrow-icon">arrow_forward</mat-icon> {{manipulation.to}}
             </span>
+            </a>
+          </div>
+        </div>
+        <div>
+           <a (click)="showAllManipulations = true;" [hidden]="showAllManipulations" style="cursor: pointer; padding-top: 10px">
+          ...
           </a>
         </div>
+        <div style="padding-top: 10px; padding-bottom: 10px;">
+        <span style="color: rgba(0, 0, 0, 0.54); font-size: 12px;">↑ up-regulation, ↓ down-regulation<mat-icon
```

Comment:
Why do the styles go in directly in the html here and not css classes?

Reply:
Because I am an idiot and I forgot it hahaha